pkg/nuctl/command/common/importreport.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2023 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/nuclio/logger"
+)
+
+type ProjectReports struct {
+	Report
```

Comment:
This isn't needed if the struct implements the interface's functions.
Same applies to rest of the reports below.

Reply:
@TomerShor why not passing explicitely?

Reply:
It's enough for a struct to have methods of an interface to implement that interface.
As in this example:
```golang
type Something interface {
	Read() string
}

type Else struct {
	name string
}

func (e *Else) Read() string {
	return e.name
}
```

`Else` implements `Something` without having an explicit reference to the interface as a property.

Reply:
@TomerShor yes, but in that case if we forget to implement some methods, we won't have any information about that during building.

Reply:
Check out these links:
- https://go.dev/doc/faq#guarantee_satisfies_interface
- https://stackoverflow.com/questions/33089523/how-to-mark-golang-struct-as-implementing-interface

I think what happens here is "an anonymous variable that implements interface". 
Without it, what information will we have if we do forget to implement?

Reply:
@TomerShor as far I remember, when once I forgot to implement a method from an interface, lint raised an error. But I've just tried to reproduce that and it's indeed do not raise it anymore. Will remove then